auth_admin_passkey/view/res_config_view.xml

```diff
@@ -17,20 +17,20 @@
 <!--You should have received a copy of the GNU Affero General Public License-->
 <!--along with this program.  If not, see <http://www.gnu.org/licenses/>.   -->
 <!-- ********************************************************************** -->
-<openerp>
+<odoo>
     <data>
 
         <record id="view_res_config_settings" model="ir.ui.view">
             <field name="name">base.config.settings.view</field>
             <field name="model">base.config.settings</field>
             <field name="inherit_id" ref="base_setup.view_general_configuration"/>
             <field name="arch" type="xml">
-                <xpath expr="//label[@string='Email']/.." position='after'>
+                <xpath expr="//label[@name='email_label']/.." position='after'>
                     <group>
                         <label for="id" string="Passkey"/>
                         <div>
                             <div>
-                                <field name="auth_admin_passkey_send_to_admin" class="oe_inline"/>
+                                <field name="" class="oe_inline"/>
```

Comment:
replace this line with:
<field name="auth_admin_passkey_send_to_admin" class="oe_inline"/>